packages/payment-processor/test/payment/erc20-escrow-payment.test.ts

```diff
@@ -0,0 +1,331 @@
+      ).toBeTruthy();
+    });
+    it('Should withdraw funds and pay funds from escrow to payee', async () => {
+      // Set a new requestID to test independent unit-tests.
+      const request = Utils.deepCopy(validRequest) as ClientTypes.IRequestData;
+      request.requestId = 'aabb';
+
+      // Execute payEscrow
+      await payEscrow(request, wallet, undefined, undefined);
+
+      // Stores balance after payEscrow(), and before withdraws.
+      const payeeBeforeBalance = await getErc20Balance(request, paymentAddress);
+      const escrowBeforeBalance = await getErc20Balance(request, escrowAddress);
+
+      await payRequestFromEscrow(request, wallet);
+
+      // Stores balances after withdraws to compare before balance with after balance.
+      const payeeAfterBalance = await getErc20Balance(request, paymentAddress);
+      const escrowAfterBalance = await getErc20Balance(request, escrowAddress);
+
+      // Expect escrow Erc20 balance should be lower.
+      expect(
+        BigNumber.from(escrowAfterBalance).eq(BigNumber.from(escrowBeforeBalance).sub(100)),
+      ).toBeTruthy();
+      // Expect payee ERC20 balance should be higher.
+      expect(
+        BigNumber.from(payeeAfterBalance).eq(BigNumber.from(payeeBeforeBalance).add(100)),
+      ).toBeTruthy();
+    });
+    it('Should initiate emergency claim', async () => {
+      // Set a new requestID to test independent unit-tests.
+      const request = Utils.deepCopy(validRequest) as ClientTypes.IRequestData;
+      request.requestId = 'aacc';
+
+      // Assign the paymentAddress as the payee.
+      const payee = getSigner(provider, paymentAddress);
+
+      // Execute payEscrow.
+      expect(
+        await (await payEscrow(request, wallet, undefined, undefined)).wait(1),
+      ).toBeTruthy();
+
+      // Payer initiate emergency claim.
+      const tx = await initiateEmergencyClaim(request, payee);
+      const confirmedTx = await tx.wait(1);
+      
+      // Checks the status and tx.hash.
+      expect(confirmedTx.status).toBe(1);
+      expect(tx.hash).not.toBeUndefined();
+    });
+    it('Should revert emergency claim', async () => {
+      // Set a new requestID to test independent unit-tests.
+      const request = Utils.deepCopy(validRequest) as ClientTypes.IRequestData;
+      request.requestId = 'aadd';
+
+      // Assign the paymentAddress as the payee.
+      const payee = getSigner(provider, paymentAddress);
+
+      // Execute payEscrow.
+      await (await payEscrow(request, wallet, undefined, undefined)).wait(1);
+
+      // Payer initiate emergency claim.
+      await (await initiateEmergencyClaim(request, payee)).wait(1);
+      
+      // Payer reverts the emergency claim.
+      const tx = await revertEmergencyClaim(request, wallet);
+      const confirmedTx = await tx.wait(1);
+
+      // Checks the status and tx.hash.
+      expect(confirmedTx.status).toBe(1);
+      expect(tx.hash).not.toBeUndefined();
```

Comment:
```suggestion
      expect(tx.hash).toBeDefined();
```
Easier to read :)

There are a few occurrences like this.

Reply:
Updated all occurrences. :-)